src/app/admin/volunteers/volunteers-details/volunteers-details.component.html

```diff
@@ -1,17 +1,8 @@
-<div class="mt-4 mb-3">
-  <a href="#" appGoBack>
-    <-
-    <span translate>back</span>
-  </a>
-</div>
-
-<div class="d-flex mb-4 align-items-center">
-  <h2 translate>view_volunteer</h2>
-  <a routerLink="edit" mat-stroked-button color="primary" class="ml-auto">
-    <mat-icon>border_color</mat-icon>
-    {{ "edit" | translate }}
-  </a>
-</div>
+<app-admin-page-header
+  *ngIf="volunteer$ | async as volunteer"
+  title="view_volunteer"
+  [editLink]="['/admin/volunteers', volunteer._id, 'edit']"
```

Comment:
I think this can be simplified to, because it will take parent route as base
```suggestion
  editLink="edit"
```